### ISSUE TITLE
saving  a text only file with "collapsed s-expr's" saves as wxme gracket format

### ISSUE THREAD

**Issue by spdegabrielle** [racket/drracket#303]
saving  a text only file with "collapsed s-expr's" saves as wxme gracket format

this was an issue for a new user 
https://groups.google.com/d/msg/racket-users/AWNG19cyD7M/sZVo8VvNEAAJ

A possible solution could be optionally un-collapsing s-exprs before save.

**Comment by spdegabrielle** [comment]
@rfindler Should this be labeled as 'good first issue'?

**Comment by rfindler** [comment]
I agree it would be nice if it was easier for people to figure out what was
going on somehow, but I am unsure how to achieve it.

I think that adding a pop up dialog is not a good solution and discarding
the collapsed state automatically is not a good idea.

Other ideas?

**Comment by samth** [comment]
I think most other editors/IDEs don't save collapsed state in the file, which is probably why someone was surprised by this behavior.

**Comment by spdegabrielle** [comment]
How about ‘save’ checking for git files if saving wxme and providing a warning when using both git and wxme?

**Comment by rfindler** [comment]
I'm hoping we can find something without warning popups. They are pretty annoying.

We could have something along the bottom of the window that indicated the file was going to be saved in WXME format instead of text format (the same way we indicate CRLF save format or if the editor is in overwrite mode or other things along those lines).

**Comment by spdegabrielle** [comment]
There is that usually empty ide status area. 
‘Good first issue’ ?

**Comment by rfindler** [comment]
I think that would be a good first issue, yes, as there is a number of things that already do similar things to be a guide. All of the code would actually be in racket/gui, but perhaps it is still better to leave this one here?
